system/autoware_component_monitor/src/component_monitor_node.cpp

```diff
@@ -147,6 +147,7 @@ std::uint64_t ComponentMonitor::parse_memory_res(const std::string & mem_res)
 {
   // example 1: 12.3g
   // example 2: 123 (without suffix, just bytes)
+  // NOLINTNEXTLINE(readability/casting)
   static const std::unordered_map<char, std::function<std::uint64_t(double)>> unit_map{
```

Comment:
This was mis-linted.
It confused `std::function<std::uint64_t(double)>>` from here `std::uint64_t(double)` part as something that was casted but it is a function definition.

So I added `// NOLINTNEXTLINE(readability/casting)` above.